vue.config.js

```diff
@@ -17,7 +17,29 @@ module.exports = {
           "https://ppng.herokuapp.com"
         ]),
       })
-    ]
+    ],
+    module: {
+      rules: [
+        // (base: https://vuetifyjs.com/en/customization/sass-variables#variable-api)
+        // SASS has different line endings than SCSS
+        // and cannot use semicolons in the markup
+        {
+          test: /\.scss$/,
+          use: [
+            'vue-style-loader',
+            'css-loader',
+            {
+              loader: 'sass-loader',
+              // Requires sass-loader@^7.0.0
+              options: {
+                // This is the path to your variables
+                data: "@import '@/styles/variables.scss'"
```

Comment:
You don't need any of this, vue-cli-plugin-vuetify does it automatically if that file exists.

Reply:
@KaelWD Thank you very much! I didn't expect you see my changes.

Reply:
It works. Thanks